omod/src/main/webapp/admin/dictionary/conceptForm.jsp

```diff
@@ -815,6 +820,65 @@
 			</table>
 		</td>
 	</tr>
+	<c:if test="${fn:substring(openmrsVersion, 0, 5) >= '2.7.0'}">
```

Comment:
Can we shield the client from this brittle logic by having something along these lines:
https://github.com/openmrs/openmrs-module-xforms/blob/master/api/src/main/java/org/openmrs/module/xforms/util/XformsUtil.java#L841

https://github.com/search?q=org%3Aopenmrs+isOnePointNineOneAndAbove&type=code

Reply:
Then you can just have a test in the jsp like this: https://github.com/search?q=repo%3Aopenmrs%2Fopenmrs-module-xforms%20useOpenmrsMessageTag&type=code